arcgis-ios-sdk-samples.xcodeproj/project.pbxproj

```diff
@@ -1259,6 +1237,7 @@
 		003D256D2513F4B6007527C2 /* AddENCExchangeSet.storyboard */ = {isa = PBXFileReference; lastKnownFileType = file.storyboard; path = AddENCExchangeSet.storyboard; sourceTree = "<group>"; };
 		003D256F2513FBB5007527C2 /* ExchangeSetWithoutUpdates */ = {isa = PBXFileReference; lastKnownFileType = folder; path = ExchangeSetWithoutUpdates; sourceTree = "<group>"; };
 		005507FB25E48134003990D3 /* Redlands.nmea */ = {isa = PBXFileReference; fileEncoding = 4; lastKnownFileType = text; path = Redlands.nmea; sourceTree = "<group>"; };
+		005B526F262E5B0800F044FE /* ArcGIS.xcframework */ = {isa = PBXFileReference; lastKnownFileType = wrapper.xcframework; name = ArcGIS.xcframework; path = $HOME/arcgis-runtime-ios/ArcGIS.xcframework; sourceTree = "<absolute>"; };
```

Comment:
I'm not sure if I'm doing it correctly here. If I remain the path unchanged as `$HOME/Library/SDKs/ArcGIS/Frameworks/ArcGIS.xcframework`, then it would give a build error:

> /Users/<<MY ID>>/<<PATH TO>>/arcgis-runtime-samples-ios/arcgis-runtime-toolkit-ios/Sources/ArcGISToolkit/AR/ArcGISARView.swift:17:8: No such module 'ArcGIS'

I think it is due to toolkit also depends on `ArcGIS.xcframework` local SP, thus both the Sample Viewer and Toolkit must have the same path.